feat(react-scheduler): add grouping by date (#2630)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/all-day-panel.md

------|------|------------
startDate | Date | The cell's start time.
endDate | Date | The cell's end time.

> groupingInfo? | Array&lt;[Group](./grouping-panel.md/#group)&gt; | Information about the cell's group(s).

Reviewer: ```suggestion
groupingInfo? | Array&lt;[Group](./grouping-panel.md/#group)&gt; | Information about the cell's grouping.
```

Reviewer: Please make the same change for every `groupingInfo` description that involves cell.

> endOfGroup? | boolean | Specifies whether it's the last cell in a group.

Reviewer: ```suggestion
endOfGroup? | boolean | `true` if this cell is last in its group.
```

Reviewer: Please make the same change for every `endOfGroup` description.
